### PR TITLE
Feature/preview appearance

### PR DESCRIPTION
**Pull request by ybs32**

### CODE REVIEW

**Comment by ybs96** [comment]
SonarQube analysis reported 24 issues
* ![MINOR][MINOR] 24 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [activities#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/activities#): Add a 'package-info.java' file to document the 'activities' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [preference#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/activities/preference#): Add a 'package-info.java' file to document the 'preference' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [array#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/adapters/array#): Add a 'package-info.java' file to document the 'array' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [holder#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/adapters/holder#): Add a 'package-info.java' file to document the 'holder' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [pager#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/adapters/pager#): Add a 'package-info.java' file to document the 'pager' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [recycler#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/adapters/recycler#): Add a 'package-info.java' file to document the 'recycler' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [application#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/application#): Add a 'package-info.java' file to document the 'application' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [fragments#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/fragments#): Add a 'package-info.java' file to document the 'fragments' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [dialog#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/fragments/dialog#): Add a 'package-info.java' file to document the 'dialog' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [preference#](https://github.com/ybs32/TweetMate-for-Android/blob/3b62b6aa1b01d823ea75174604ae60dc280cf7a6/app/src/main/java/com/ybsystem/tweethub/fragments/preference#): Add a 'package-info.java' file to document the 'preference' package [rule](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1228)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by ybs32** [comment]
@ybs96 
Thanks for the code review.
Sonar settings may not be configured for Android. (needs investigation)

File: app/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+public class PreviewAppearanceDialog extends DialogFragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate
+        View view = inflater.inflate(R.layout.dialog_preview_appearance, null);
+
+        // Create timeline
+        Fragment fragment = new ProfileTimeline().newInstance(
+                new Column(7080152L, "", USER_TWEET, false),
```
Comment (ybs32) [inline app/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java:27]: This "7080152L" is a magic number. (Self-review)

Comment (ybs32) [reply app/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java]: Fixed: https://github.com/ybs32/TweetMate-for-Android/pull/67/commits/553f65ad879d79416490cc616857fe7f7a43b652

**Pull request merged**